### PR TITLE
Prep for distro

### PR DESCRIPTION
**Pull request by shacker**
Includes "final" original_images and data fixture, with readme tweaks.

### CODE REVIEW

**Comment by shacker** [comment]
@heymonkeyriot Please try:

- rename your bakerydemodb
- migrate up a fresh db 
- `./manage.py load_initial_data`

and let me know if everything is working.

 I don't have postgres working yet (coming), but loading up a fresh sqlite seems fine from this end.

File: readme.md [hunk 1]
```diff
@@ -74,10 +74,11 @@ To tail the logs from the Docker containers in realtime, run:
 docker-compose logs -f
 ```
 
-Local Setup
------------
-Don't want to set up a whole Vagrant VM or use Docker to try out Wagtail? No problem. You'll probably want to start
-with a fresh virtualenv.
+Traditional (Local) Setup
```
Comment (heymonkeyriot) [inline readme.md]: I think we should be called this "Virtualenv setup"

File: readme.md [hunk 2]
```diff
@@ -74,10 +74,11 @@ To tail the logs from the Docker containers in realtime, run:
...
+-----------------------
+
+Want to run the bakerydemo locally without setting up Vagrant or Docker? You'll probably want to start
+with a fresh [virtualenv](http://docs.python-guide.org/en/latest/dev/virtualenvs/).
```
Comment (heymonkeyriot) [inline readme.md]: Under 'Setup with Vagrant' we have

```
### Dependencies
* [VirtualBox](https://www.virtualbox.org/)
* [Vagrant 1.5+](http://www.vagrantup.com)
```

I'd suggest, for consistency, that we follow the same pattern. e.g
```
```
### Dependencies
* [Virtualenv](https://virtualenv.pypa.io/en/stable/installation/)
* [VirtualenvWrapper](http://virtualenvwrapper.readthedocs.io/en/latest/install.html)
```

File: readme.md (hunk 1 above)
Comment (shacker) [inline readme.md]: changing this now

File: readme.md (hunk 2 above)
Comment (shacker) [inline readme.md]: Done.

**Pull request merged**